src/licensevalidator/lib/workflowlicenses.py

```diff
@@ -159,7 +162,7 @@ def __get_license_for_action(action_repo: str, github_token: str) -> Optional[st
     Returns:
         str: The name of the license, if available.
     """
-    request_headers={"Accept": "application/vnd.github.v3+json"}
+    request_headers = {"Accept": "application/vnd.github.v3+json"}
```

Comment:
in all other project we force the space between the assignments, please adjust the settings accordingly